condition_resource_contains.go

```diff
@@ -0,0 +1,41 @@
+package ladon
+
+import (
+	"regexp"
+)
+
+// ResourceContainsCondition is fulfilled if the context matches a substring within the resource name
+type ResourceContainsCondition struct{}
+
+// Fulfills returns true if the request's resouce contains the given value string
+func (c *ResourceContainsCondition) Fulfills(value interface{}, r *Request) bool {
+
+	filter, ok := value.(map[string]interface{})
+	if !ok {
+		return false
+	}
+
+	valueString, ok := filter["value"].(string)
+	if !ok || len(valueString) < 1 {
+		return false
+	}
+
+	//If no delimiter provided default to "equals" check
+	delimiterString, ok := filter["delimiter"].(string)
+	if !ok || len(delimiterString) < 1 {
+		delimiterString=""
+	}
+
+	// Append delimiter to strings to prevent delim+1 being interpreted as delim+10 being present
+	filterValue := delimiterString + valueString + delimiterString
+	resourceString := r.Resource + delimiterString
```

Comment:
This will not work if the resource is `foo:bar`, delimiter is `:`, and the filter value is also `foo:bar`. Add this as a failing test case before working on it.

Reply:
Will have a look on it tomorrow, 
Note to self: an exakt string equals check before the contains check, may solve this.